tests/EventStream.Tests/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "EventStream.Tests Class Library",
+  "authors": [ "Stuart Blackler" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+  "dependencies": {
+    "EventStream": "1.0.0-*",
+    "FluentAssertions": "4.0.0",
+    "Microsoft.Framework.DependencyInjection": "1.0.0-beta8",
+    "xunit": "2.1.0",
```

Comment:
Reckon you can throw in TestAttributes so that this and #13 can merge concurrently?